app/models/user/UserCurrentRegionTable.scala

```diff
@@ -64,18 +64,46 @@ object UserCurrentRegionTable {
     val currentRegionList = _currentRegions.list
 
     if (currentRegionList.isEmpty) {
+      // For a new user whose current region is not assigned, assign an easy least audited region
+      assignEasyRegion(userId)
+    } else {
+      // Note: Which case would this be? assignRandomly() is only called immediately after signing up or when the
```

Comment:
Not sure... It doesn't seem like this should happen. So I would probably log a warning, and call assignNextRegion, like you did.